ansible/roles/fanout/tasks/fanout_sonic.yml

```diff
@@ -21,3 +21,8 @@
   include_tasks:
     sonic/fanout_sonic_202012.yml
   when: "'20201231' in fanout_sonic_version['build_version']"
+
+- name: deploy SONiC fanout with image version master
+  include_tasks:
+    sonic/fanout_sonic_202012.yml
```

Comment:
Do we need to create a new file named `fanout_sonic_master.yml`?

Reply:
master is not stable enough, let's use `202205` instead

Reply:
Updated to `202205`